schedule/yast/textmode/textmode_x86_64.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: textmode
+description: |
+  Installation in textmode which selects System Role 'Text Mode'.
+vars:
+  YUI_REST_API: 1
+schedule:
+  system_preparation:
+    - console/consoletest_setup
+  system_validation:
+    - console/validate_product_installed_SLES
+    - console/verify_network
+    - locale/keymap_or_locale
+    - console/validate_installed_patterns
+    - console/force_scheduled_tasks
```

Comment:
looks like the test are not in order in the original job, could you order them, please? this one looks preparation not validation.

Reply:
changed and ordered.